app-directory/share-state/.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "typescript.tsdk": "node_modules\\typescript\\lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true
+}
```

Comment:
Do we need this file?

Reply:
Is added automatically when you agree to use the Next.js plugin, we can add the `.vscode` folder to gitignore if you want